source/admin_manual/list_of_events.rst

```diff
@@ -164,646 +164,284 @@ SQL Categories
 +--------------------+---------------------------------------------------------+
 
 
-*************
-Global Fields
-*************
-
-**ALL events may have the following fields**:
-
-+--------------+------------------------------------------------------------+
-| Field        | Description                                                |
-+==============+============================================================+
-| duration     | Duration of the event (in microseconds)                    |
-+--------------+------------------------------------------------------------+
-| reason_code  | List of reason code strings why the event happened. See    |
-|              | :ref:`event_reasons` for possible values.                  |
-+--------------+------------------------------------------------------------+
+******
+Events
+******
 
+.. dovecot_event:field_global::
 
-************
-Dovecot Core
-************
+   :field duration: Duration of the event (in microseconds)
+   :field reason_code: List of reason code strings why the event happened. See
+                       :ref:`event_reasons` for possible values.
 
 Authentication Client
 =====================
 
-auth_client_cache_flush_started
--------------------------------
+These events are generated by authentication clients (lib-auth).
 
-*no particular fields*
+.. dovecot_core:event:: auth_client_cache_flush_started
+   :added: v2.3.7
```

Comment:
I think we should just remove this. I don't see any use for this. Same with _finished. This event is currently sent by "doveadm auth cache flush" command. It could make sense to have "auth_cache_flush_finished" event on the auth server side, but it makes no sense to have a client-side event.

Reply:
DOP-2869, we should remove in 3.0.  I'll mark as deprecated/removed in this MR and just put placeholder text for the TODO.